emoji/src/main/java/com/vanniktech/emoji/RecentVariantManager.java

```diff
@@ -0,0 +1,145 @@
+package com.vanniktech.emoji;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.support.annotation.NonNull;
+import com.vanniktech.emoji.emoji.Emoji;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.StringTokenizer;
+
+final class RecentVariantManager { // TODO: test it
```

Comment:
Thought about it and I don't think that you want to do this. If there's high demand for it we can still expose it. I prefer to keep as many internals internals.